addons/web/static/src/js/views/graph/graph_controller.js

```diff
@@ -130,16 +130,16 @@ var GraphController = AbstractController.extend({
                 items: this.measures,
             });
             this.buttonDropdownPromises = [this.measureMenu.mount(fragment)];
+            if (this.isEmbedded) {
```

Comment:
done for dashboard (the buttons are rendered without node and then one tries to append the groupByMenu without having created it)

Reply:
where is the fragment appendend to dom ?
in on_attach_callback ?

Reply:
A node is provided to renderButtons in _renderSubViewButtons (DashboardRenderer)
but renderButtons and updateButtons are called before that when instantiating the graph subview (AbstractController).
There the node is not given and when the update occurs, the groupByMenu has not been 'started' in a fragment so that its el is undefined.